setup.py

```diff
@@ -35,9 +38,6 @@
 with open("pdb2pqr/_version.py") as fobj:
     exec(fobj.read())
 
-if version_info[:2] < (3, 8):
```

Comment:
Why was version checking removed?

Reply:
This is a wrong way of defining minimum version. The info should go to `python_requires` field in `setup()`. If you put it as a conditional, `pip install` will have a wrong behaviour (just ignoring them).

See https://github.com/neovim/pynvim/issues/582#issuecomment-2470708148, for example.